### PR TITLE
Add add_dns_entries function on DomainService.

### PR DESCRIPTION
**Pull request by daramousk**

### CODE REVIEW

**Comment by roaldnefs** [comment]
@benkonrath, what do you think of this addition? I thinks this is a good addition, I have already heard several people who are annoyed by the fact that you have to include all existing DNS entries. This function makes it more user-friendly but is not part of the TransIP API.

**Comment by benkonrath** [comment]
I think it's a good addition. :+1: I've had to write some code like this in an application but I think it makes sense to just have it in the library.

I could see a helper for remove and update also being useful - I haven't looked at the complexities, I'm just thinking as an application developer. To be clear: remove & update aren't required for this change to be merged.

File: transip/service/domain.py [hunk 1]
```diff
@@ -152,6 +152,15 @@ def set_dns_entries(self, domain_name, dns_entries):
         """
         return self._simple_request('setDnsEntries', domain_name, dns_entries, mode=MODE_RW)
 
+    def add_dns_entries(self, domain_name, dns_entries):
+        """
+        Adds the given DnsEntries to the domain.
+        :type domain_name str
```
[the remark this answers is not in the document]
Comment (roaldnefs) [reply transip/service/domain.py]: I've updated the docstring  in https://github.com/benkonrath/transip-api/pull/48/commits/1420e2e3ff290dcd38ca8da054c0cd77bfe4c3d8.

**Comment by roaldnefs** [comment]
@daramousk thanks for your contribution!

If you want to take a stab at the suggestions of @benkonrath please let us know.

**Comment by daramousk** [comment]
Hey Roald and Ben, you are welcome, I will be pushing some extras as the requirements come in, I have already pushed another one.

Thanks for the quick response.

**Pull request merged**